units/patch_intent.go

```diff
@@ -159,6 +159,16 @@ func (j *patchIntentProcessor) Run(ctx context.Context) {
 }
 
 func (j *patchIntentProcessor) finishPatch(ctx context.Context, patchDoc *patch.Patch) error {
+	// Add owner and repo for child patches
+	if patchDoc.IsChild() {
```

Comment:
We should probably move this logic to where the patch intent is created for child patches

Reply:
looking at the code, FinalizePatch might be a good place for it since we already have an isChild check and access to the project.

Reply:
finalize patch wouldnt hit certain use cases i think like backport but i moved it out to running patch intent processor